korman/exporter/material.py

```diff
@@ -721,12 +721,24 @@ def _propagate_material_settings(self, bm, layer):
             state.shadeFlags |= hsGMatState.kShadeNoFog # Dead in CWE
             state.shadeFlags |= hsGMatState.kShadeReallyNoFog
 
+        if bm.use_shadeless:
+            state.shadeFlags |= hsGMatState.kShadeWhite
+
         # Colors
         layer.ambient = utils.color(bpy.context.scene.world.ambient_color)
         layer.preshade = utils.color(bm.diffuse_color)
         layer.runtime = utils.color(bm.diffuse_color)
         layer.specular = utils.color(bm.specular_color)
 
+        if bm.emit > 0.0:
+            # Use the diffuse colour as the emit, scaled by the emit amount
+            # (maximum 2.0, so we'll also scale that by 0.5)
```

Comment:
I'm not sure I understand this?

Reply:
Blender Material has an emit slider whose range is apparently [0.0, 2.0]. So a value of 1.0 appears to be 50%, when a multiplication would interpret that as 100%.